### ISSUE TITLE
Duplicately named nodes (DOID and phenot_phenotype)

### ISSUE THREAD

**Issue by jaredroach** [RTXteam/RTX#67]
duplicate disease / phenotype  (DOID and phenot_phenotype) are same thing. E.g.,

<id>:160027UUID: 5ce16ebe-d3f5-11e7-82ef-0277022df40c
description: Multiple myeloma
expanded: truename: HP:0006775
&
<id>:150835UUID: 0e813992-d400-11e7-82ef-0277022df40c
description: multiple myeloma
expanded: truename: DOID:9538

Not sure the duplication is useful. Maybe delete one of these nodes? I treat them interchangeably when constructing gold standard answers.

**Comment by dkoslicki** [comment]
This was also noticed in #60. The way I've been handling it is to only accept disease nodes as terminal end points to the COP's. This is in line with Eric's request that each query type be associated to fixed node types. So if we want phenotypes as endpoints to the COP's as well, this could be implemented (but would require a new solution script specific to this as then I would be looking for paths ending at different node types).

If you want COP's to allow phenotype endpoints, let me know and I can implement it.

**Comment by jaredroach** [comment]
I would prefer to delete from the KG phenotypes which are duplicates of disease nodes.

**Comment by dkoslicki** [comment]
Maybe @saramsey can weigh in: is it reasonable to keep phenotype nodes that have a description identical to a disease? I know keeping as many phenotype nodes as possible does help with answering some of the questions (such as "what are the symptoms of osteoarthritis"). Deleting the phenotype nodes would make answering these kinds of questions more difficult (since then I can't just look for the node type ``phenotypic_feature`` via relationship ``has_phenotype``).

**Comment by saramsey** [comment]
Yes, the reason to keep them is that it would create complications (in terms of KG construction) to delete them.  Complications that could be surmounted with effort, sure.  But what is to be gained (from a practical standpoint in terms of RTX functionality) by deleting them?

-----------------------------------------------------
Stephen Ramsey
Assistant Professor, Oregon State University
* School of Electrical Engineering and Computer Science
* Department of Biomedical Sciences
208A Dryden Hall
stephen.ramsey@oregonstate.edu<mailto:stephen.ramsey@oregonstate.edu>
http://lab.saramsey.org







On Apr 16, 2018, at 2:33 PM, David Koslicki <notifications@github.com<mailto:notifications@github.com>> wrote:


Maybe @saramsey<https://github.com/saramsey> can weigh in: is it reasonable to keep phenotype nodes that have a description identical to a disease? I know keeping as many phenotype nodes as possible does help with answering some of the questions (such as "what are the symptoms of osteoarthritis"). Deleting the phenotype nodes would make answering these kinds of questions more difficult (since then I can't just look for the node type phenotypic_feature via relationship has_phenotype).

**Comment by saramsey** [comment]
I have no objection to keeping this as a low-priority issue, however. Am just thinking it may not be worth implementing this issue, before the Hackathon.

**Comment by dkoslicki** [comment]
Agreed, tabled until after the hackathon.
